File: programitas/Primos.java
import java.util.Date;
public class Primos {
	public static void main(String args[]){
		long ini = new Date().getTime();
		int primo = 2;
		while (primo <= 150000){
			int i = 1;
			int dividen = 0;
			while(i <= primo){
				if(primo % i == 0){
					dividen++;
				}
				i++;
			}
			if(dividen == 2){
				System.out.println("Primo encontrado: " + primo);
			}
			primo++;
		}
		long fin = new Date().getTime();
		System.out.println("El tiempo de ejecucion fue: " + (fin - ini) + " milisegundos");
	}
}

File: programitas/primos.cpp
#include<iostream>
#include<ctime>
/*
Programa que imprime los numeros primos hasta el intervalo que se le indique
ademas encuentra el tiempo de ejecucion.
*/
using namespace std;

int main (){
	time_t ini = clock();
	int primo = 2;
	while (primo <= 15000){
		int i = 1;
		int dividen = 0;
		while(i <= primo){
			if(primo % i == 0){
				dividen++;
			}
			i++;
		}
		if(dividen == 2){
			cout << "Primo encontrado: " << primo << endl;
		}
		primo++;
	}
	ini = clock() - ini;
	cout << "El tiempo total de ejecucion fue " << (float)ini/100 << " milisegundos" << endl; 
	return 0;
}

